documentation/api/livemetrics-custom.md

```diff
@@ -146,7 +146,7 @@ Location: localhost:52323/operations/67f07e40-5cca-4709-9062-26302c484f18
 
 {
     "timestamp": "2021-08-31T16:58:39.7514031+00:00",
-    "provider": "CustomProvider",
+    "provider": "CustomMeter",
```

Comment:
I didn't think of this before, but worth considering before we lock ourselves into a schema - do we want to use `meter` instead of `provider` if we know we're using an SDM?

Reply:
This is likely to lead to a lot more unnecessary special casing. While I want to split the configuration, I'm not sure I want to split the data model so much.